benchmarks/src/jmh/kotlin/benchmarks/flow/FlowFlattenMergeBenchmark.kt

```diff
@@ -56,6 +54,23 @@ open class FlowFlattenMergeBenchmark {
     }
 }
 
+enum class Flows {
+    `10xConcurrency flows` {
+        override fun getFlows(concurrency: Int): Int = 10 * concurrency
+    },
+    `1xConcurrency flows` {
+        override fun getFlows(concurrency: Int): Int = concurrency
+    },
+    `100 flows` {
+        override fun getFlows(concurrency: Int): Int = 100
+    },
+    `500 flows` {
+        override fun getFlows(concurrency: Int): Int = 500
+    };
+
+    abstract fun getFlows(concurrency : Int): Int
```

Comment:
It is simpler to add the corresponding lambda to the constructor, e.g. `val get: (concurrency: Int) -> Int`. Thus, your code becomes shorter and easier to read.